packages/@aws-cdk/core/lib/resolvable.ts

```diff
@@ -17,6 +17,11 @@ export interface IResolveContext {
    */
   readonly preparing: boolean;
 
+  /**
+   * Path
```

Comment:
Is this the path in the construct tree? Can you explain it further? Or should that be obvious?

Reply:
It's the path in the resulting JSON document. I'll clarify.